### DISCUSSION TITLE
Moving "The" to end of name.. IE: title of movie, the

### DISCUSSION THREAD

**Discussion by geekm0nkey** [jkwill87/mnamer#157] [Ideas and Requests]
Hello there.. First, thank you for this application. It is very handy.. 
My request comes from wanting to have all my file sorted by name, sorting by the word "The" is awkward.  Moving it to the end allows the movies to be sorted more properly IMHO.. such as..

Matrix, The
Matrix II, The

Etc... Is this possible to do with MNAMER? I was unable to locate the option in the formatting..

Thank you.

**Comment by jkwill87** [comment]
This isn't something that is currently supported but I think it would be reasonable to add. Thanks for the suggestion.
